Documentation/gettingstarted/istio.rst

```diff
@@ -455,56 +464,56 @@ whitelisted:
 
 .. literalinclude:: ../../examples/kubernetes-istio/bookinfo-productpage-v2-policy.yaml
 
-Create the ``productpage v2`` service and its updated
-CiliumNetworkPolicy and delete ``productpage v1``:
-
-.. parsed-literal::
+Because ``productpage v2`` sends messages into Kafka, we must first
```

Comment:
This section is too long, I think a subsection would be better here "Logging unauthorized requests in Kafka" or something like that